src/layouts/dialog.vue

```diff
@@ -16,33 +16,29 @@ export default Vue.extend({
     // the "dark" part will be a dynamic pref.
     // See https://github.com/rancher/dashboard/blob/3454590ff6a825f7e739356069576fbae4afaebc/layouts/default.vue#L227 for an example
     return { bodyAttrs: { class: 'theme-dark' } };
-  },
-  mounted() {
-    const wrapper = this.$refs.wrapper as HTMLDialogElement;
-
-    // Dynamically resize the window to fit the contents.
-    // We need a bit extra on the width to ensure we don't get scroll bars.
-    window.resizeBy(
-      wrapper.offsetWidth - document.documentElement.offsetWidth + 14,
-      wrapper.offsetHeight - document.documentElement.offsetHeight);
   }
 });
 </script>
 
+<style lang="scss">
+  html {
+    height: initial;
+  }
+</style>
```

Comment:
I do not like overriding global styles here, but this was the simplest way to accurately report height of our contents via the `preferred-size-changed` event. I'm open to alternative implementations.